pkg/ts/core/Particle.ts

```diff
@@ -226,6 +226,7 @@ export class Particle {
     this.pipe = pipe;
     this.impl = create(proto);
     defineProperty(this, 'internal', privateProperty(nob()));
+    this.internal.$busy = 0;
```

Comment:
What's a '$' prefix on a field in TS, is it still just the symbol name or does it have some effect?

Reply:
`$` is legal in an identifier, so there is no explicit effect. Usually we use it to denote a 'keyword' in a JSON structure, but here it's (subjectively) just a mistake. I think I intended this to mean `private`, but doesn't `internal` do that?